client/src/components/leftPanel/LeftPanel.js

```diff
@@ -6,37 +6,58 @@ import './LeftPanel.scss'
 class LeftPanel extends Component {
 
   state = {
-    searchString: ""
+    searchString: "",
+    showArchived: false,
   }
 
   handleChange = (event) => this.setState({ searchString: event.target.value })
   
+  onShowArchivedChange = (event) => this.setState({showArchived: event.target.checked})
+
   render() {
     const {tags} = this.props
-    const {searchString} = this.state
+    const {searchString, showArchived} = this.state
     const lowerCaseSearchString = searchString.trim().toLowerCase();
 
     return (
-      <div className="LeftPanel">
-        <a href='/'>
-          <img className="logo" src={vacuumLogo} alt="VacuumLabs logo" />
-        </a>
-        <strong className="sectionTitle">Select tag</strong>
-        <input
-          type="text"
-          value={searchString}
-          onChange={this.handleChange}
-          placeholder="search"
-        />
-        <ul>
-          {tags
-            .filter(({tag}) => tag.toLowerCase().includes(lowerCaseSearchString))
-            .map((tag) => (<Tag key={tag.tag} tag={tag} />)
-          )}
-        </ul>
+      <div className={`LeftPanelContainer${searchString ? '' : ' padded'}`}>
+        <div className="LeftPanel">
+          <a href='/'>
+            <img className="logo" src={vacuumLogo} alt="VacuumLabs logo" />
+          </a>
+          <strong className="sectionTitle">Select tag</strong>
+          <input
+            type="text"
+            value={searchString}
+            onChange={this.handleChange}
+            placeholder="search"
+          />
+          {!searchString && <ShowArchivedBox label="show archived projects" isChecked={showArchived}
+                                             onChange={this.onShowArchivedChange} />}
```

Comment:
(nitpick) Is eslint forcing you to do this formatting? If not I would prefer the following:
```
{!searchString && (
	<ShowArchivedBox
		label="show archived projects"
		isChecked={showArchived}
		onChange={this.onShowArchivedChange}
	/>
)}
```